dali/test/operator_argument.h

```diff
@@ -27,6 +29,48 @@ struct InputArg {
   // TODO(mszolucha): For Dali's argument input
 };
 
+template <typename T, typename Enable = void>
+struct TestOpArgToStringImpl {
+  static std::string to_string(const T&) {
+    return {};
+  }
+};
+
+template <typename T>
+struct TestOpArgToStringImpl<T, typename std::enable_if<std::is_arithmetic<T>::value>::type> {
+  static std::string to_string(const T& val) {
+    return std::to_string(val);
+  }
+};
+
+template <typename T>
+struct TestOpArgToStringImpl<std::vector<T>,
+                             typename std::enable_if<std::is_arithmetic<T>::value>::type> {
+  static std::string to_string(const std::vector<T>& val) {
+    std::stringstream ss;
+    if (val.size() == 0) {
+      return "[]";
+    } else {
+      ss << "[ ";
+    }
+    for (size_t i = 0; i < val.size(); i++) {
+      ss << val[i];
+      if (i != val.size() - 1) {
+        ss << ", ";
+      }
+    }
+    ss << " ]";
+    return ss.str();
+  }
+};
+
+template <>
+struct TestOpArgToStringImpl<std::string> {
+  static std::string to_string(const std::string& val) {
+    return "\"" + val + "\"";
+  }
+};
+
```

Comment:
I had to comment out some of the code in this file yesterday, because it was causing trouble to clang

Reply:
It was missing `std::` in to_string specialization for integers.